pkgs/by-name/sn/snappy/package.nix

```diff
@@ -3,30 +3,22 @@
   stdenv,
   fetchFromGitHub,
   cmake,
-  fetchpatch,
   static ? stdenv.hostPlatform.isStatic,
 }:
 
 stdenv.mkDerivation rec {
   pname = "snappy";
-  version = "1.2.1";
+  version = "1.2.2";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "snappy";
     rev = version;
-    hash = "sha256-IzKzrMDjh+Weor+OrKdX62cAKYTdDXgldxCgNE2/8vk=";
+    hash = "sha256-bMZD8EI9dvDGupfos4hi/0ShBkrJlI5Np9FxE6FfrNE=";
   };
 
   patches = [
-    # Re-enable RTTI, without which other applications can't subclass
-    # snappy::Source (this breaks Ceph, as one example)
-    # https://tracker.ceph.com/issues/53060
-    # https://build.opensuse.org/package/show/openSUSE:Factory/snappy
```

Comment:
@trofi coming from

* https://github.com/NixOS/nixpkgs/issues/426401#issuecomment-3117744901

should this patch no longer be necessary?

I do not immediately understand the relation between the `revert-PUBLIC.patch` and the previous RTTI patch.